### PR TITLE
PEP8

### PR DESCRIPTION
**Pull request by larroy**
*Description of changes:*

PEP8 fixes and check for additional PEP8 warnings in CI.

Line width set to 120.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by zhanghang1989** [comment]
To enable the CI, please add flake8 to https://github.com/awslabs/autogluon/blob/master/docs/build.yml#L6

File: autogluon/utils/tabular/ml/trainer/abstract_trainer.py [hunk 1]
```diff
@@ -61,14 +66,18 @@ def __init__(self, path: str, problem_type: str, scheduler_options=None, objecti
             self.stopping_metric = self.objective_func
 
         self.objective_func_expects_y_pred = scorer_expects_y_pred(scorer=self.objective_func)
-        logger.log(25, "AutoGluon will gauge predictive performance using evaluation metric: %s" % self.objective_func.name)
+        logger.log(25, "AutoGluon will gauge predictive performance using evaluation metric: %s" %
+                   self.objective_func.name)
         if not self.objective_func_expects_y_pred:
-            logger.log(25, "This metric expects predicted probabilities rather than predicted class labels, so you'll need to use predict_proba() instead of predict()")
+            logger.log(
```
Comment (Innixma) [inline autogluon/utils/tabular/ml/trainer/abstract_trainer.py]: This is an example where the line limit makes things look ugly

File: autogluon/utils/tabular/ml/trainer/abstract_trainer.py [hunk 2]
```diff
@@ -352,16 +374,19 @@ def train_single_full(self, X_train, y_train, X_test, y_test, model: AbstractMod
...
                 if type(model) in [BaggedEnsembleModel, StackerEnsembleModel, WeightedEnsembleModel]:
-                    hpo_models, hpo_model_performances, hpo_results = model.hyperparameter_tune(X=X_train, y=y_train, k_fold=kfolds, scheduler_options=(self.scheduler_func, self.scheduler_options), verbosity=self.verbosity)
+                    hpo_models, hpo_model_performances, hpo_results = model.hyperparameter_tune(
+                        X=X_train, y=y_train, k_fold=kfolds, scheduler_options=(self.scheduler_func, self.scheduler_options), verbosity=self.verbosity)
                 else:
                     if (X_test is None) or (y_test is None):
-                        X_train, X_test, y_train, y_test = self.generate_train_test_split(X_train, y_train, test_size=0.2)  # TODO: Adjust test_size, perhaps user specified?
+                        X_train, X_test, y_train, y_test = self.generate_train_test_split(
+                            X_train, y_train, test_size=0.2)  # TODO: Adjust test_size, perhaps user specified?
                     hpo_models, hpo_model_performances, hpo_results = model.hyperparameter_tune(X_train=X_train, X_test=X_test,
```
Comment (Innixma) [inline autogluon/utils/tabular/ml/trainer/abstract_trainer.py]: This is an example where a func returns multiple outputs which could make the params take up many lines

**Comment by Innixma** [comment]
Since this is such a big update, can we split it into two parts:

Part 1: All PEP8 changes EXCEPT line character limits

Part 2: PEP8 line character limits

I am currently aligned with Part 1, and for Part 2 we should sync with the team to understand the proper character limit we want going forward.

Best,
Nick

**Comment by szha** [comment]
Job PR-272-4 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-272/4/index.html

**Pull request merged**